mongodb-quickstart/pom.xml

```diff
@@ -40,6 +40,14 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-smallrye-context-propagation</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jackson</artifactId>
```

Comment:
We should switch to RestEasy reactive (while keeping imperative code)

Reply:
I missed this one, thanks for spotting.